CHANGELOG.md

```diff
@@ -5,9 +5,10 @@
 
 ### Feature:
 * Introduce new SequentialHostInit view allocation property [\#7229](https://github.com/kokkos/kokkos/pull/7229)
+* Experimental CUDA support for unified memory mode (intended for Grace-Hopper etc.) [\#6823](https://github.com/kokkos/kokkos/pull/6823)
```

Comment:
Not sure if it deserves to be a feature considering it is an IMP_ option
Maybe "backend enhancement"?

Reply:
yeah sure

Reply:
I moved it